app/code/Magento/Deploy/Package/Processor/PreProcessor/Css.php

```diff
@@ -180,11 +180,11 @@ private function buildMap($packagePath, $filePath, $fullPath)
      */
     private function collectFileMap($fileName)
     {
-        $result = isset($this->map[$fileName]) ? $this->map[$fileName] : [];
+        $result = isset($this->map[$fileName]) ? [$this->map[$fileName]] : [[]];
```

Comment:
what do u think about Elvis operator? like this 
$result = [$this->map[$fileName]] ?? [[]];